pkg/bpf/bpffs.go

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package bpf
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/cilium/tetragon/pkg/defaults"
+	"github.com/cilium/tetragon/pkg/logger"
+	"github.com/cilium/tetragon/pkg/mountinfo"
+)
+
+var (
+	// Path to where bpffs is mounted
+	mapRoot = defaults.DefaultMapRoot
+
+	// Path to where debugfs is mounted
+	debugFSRoot = "/sys/kernel/debug"
```

Comment:
this is linux specific variable, let's move it to linux code

Reply:
I have moved this to bpffs_linux